sdk/documentintelligence/azure-ai-documentintelligence/tests/testcase.py

```diff
@@ -0,0 +1,864 @@
+
+    if get_element_type(element_pointer) == "word":
+        line = indices[1]
+        word = indices[2]
+        ocr_word = read_result[read].lines[line].words[word]
+        return "word", ocr_word, read + 1
+
+    if get_element_type(element_pointer) == "line":
+        line = indices[1]
+        ocr_line = read_result[read].lines[line]
+        return "line", ocr_line, read + 1
+
+    if get_element_type(element_pointer) == "selectionMark":
+        mark = indices[1]
+        selection_mark = read_result[read].selection_marks[mark]
+        return "selectionMark", selection_mark, read + 1
+
+    return None, None, None
+
+
+class FakeTokenCredential(object):
+    """Protocol for classes able to provide OAuth tokens.
+    :param str scopes: Lets you specify the type of access needed.
+    """
+
+    def __init__(self):
+        self.token = AccessToken("YOU SHALL NOT PASS", 0)
+
+    def get_token(self, *args, **kwargs):
+        return self.token
+
+
+class DocumentIntelligenceTest(AzureRecordedTestCase):
+
+    testing_container_sas_url = os.getenv("DOCUMENTINTELLIGENCE_TESTING_DATA_CONTAINER_SAS_URL", "https://blob_sas_url")
+    barcode_url_tif = _get_blob_url(testing_container_sas_url, "testingdata", "barcode2.tif")
+    formula_url_jpg = _get_blob_url(testing_container_sas_url, "testingdata", "formula4.jpg")
+    receipt_url_jpg = _get_blob_url(testing_container_sas_url, "testingdata", "contoso-allinone.jpg")
+    receipt_url_png = _get_blob_url(testing_container_sas_url, "testingdata", "contoso-receipt.png")
+    business_card_url_jpg = _get_blob_url(testing_container_sas_url, "testingdata", "businessCard.jpg")
+    business_card_url_png = _get_blob_url(testing_container_sas_url, "testingdata", "businessCard.png")
+    business_card_multipage_url_pdf = _get_blob_url(
+        testing_container_sas_url, "testingdata", "business-card-multipage.pdf"
+    )
+    identity_document_url_jpg = _get_blob_url(testing_container_sas_url, "testingdata", "license.jpg")
+    identity_document_url_jpg_passport = _get_blob_url(testing_container_sas_url, "testingdata", "passport_1.jpg")
+    invoice_url_pdf = _get_blob_url(testing_container_sas_url, "testingdata", "Invoice_1.pdf")
+    invoice_url_tiff = _get_blob_url(testing_container_sas_url, "testingdata", "Invoice_1.tiff")
+    invoice_url_jpg = _get_blob_url(testing_container_sas_url, "testingdata", "sample_invoice.jpg")
+    multipage_vendor_url_pdf = _get_blob_url(testing_container_sas_url, "testingdata", "multi1.pdf")
+    form_url_jpg = _get_blob_url(testing_container_sas_url, "testingdata", "Form_1.jpg")
+    multipage_url_pdf = _get_blob_url(testing_container_sas_url, "testingdata", "multipage_invoice1.pdf")
+    multipage_table_url_pdf = _get_blob_url(testing_container_sas_url, "testingdata", "multipagelayout.pdf")
+    selection_mark_url_pdf = _get_blob_url(testing_container_sas_url, "testingdata", "selection_mark_form.pdf")
+    label_table_variable_row_url_pdf = _get_blob_url(
```

Comment:
This file will also need some clean up since it's referencing variables not used in this new library.

Reply:
Agree. I think we can leave the clean up (and in other files) later as the releasing is expect to ship today.